Tests/Feature/Product/DeleteProductTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PlugAndPay\Sdk\Tests\Feature\Product;
+
+use PHPUnit\Framework\TestCase;
+use PlugAndPay\Sdk\Entity\Response;
+use PlugAndPay\Sdk\Exception\NotFoundException;
+use PlugAndPay\Sdk\Exception\UnauthenticatedException;
+use PlugAndPay\Sdk\Service\ProductService;
+use PlugAndPay\Sdk\Tests\Feature\Product\Mock\ProductDestroyMockClient;
+
+class DeleteProductTest extends TestCase
+{
+    /** @test */
+    public function delete_product_not_fount_test(): void
+    {
+        $client  = new ProductDestroyMockClient(Response::HTTP_NOT_FOUND, []);
```

Comment:
```suggestion
        $client  = new ProductDestroyMockClient(Response::HTTP_NOT_FOUND);
```
Zie ook andere tests